src/hooks/useParentBasicName.ts

```diff
@@ -1,14 +1,9 @@
-import { useBasicName } from '@app/hooks/useBasicName'
-
-export const useParentBasicName = (name: string) => {
-  const nameParts = name.split('.')
-  const parentName = nameParts.slice(1).join('.')
-
-  const isValidParent = parentName.split('.').length > 1
-
-  const validParentName = isValidParent ? parentName : ''
+import { useBasicName, UseBasicNameOptions } from '@app/hooks/useBasicName'
+import { parentName } from '@app/utils/name'
 
+export const useParentBasicName = ({ name, enabled, ...otherProps }: UseBasicNameOptions) => {
+  const parentName_ = parentName(name)
   // So far the use of this hook is for the wrapper data which does not rely on the graph so
   // it is safe to skip the graph
-  return useBasicName({ name: validParentName })
+  return useBasicName({ name: parentName_, ...otherProps })
```

Comment:
since `subgraphEnabled` was readded to `useBasicName`, should that be specified as false here?

Reply:
I removed the comment. This is no longer true anymore. replaced useBasicname(<parent-name>) with useParentBasicName so that it would be easier to mock the data in the future. I removed the comment.